src/templates/options.js

```diff
@@ -11,7 +11,7 @@ function stringifyValue(value) {
 
 for (const [key, value] of Object.entries(options)) {
     if (key === 'vueI18n' && typeof value === 'string') {
-%>export const <%= key %> = require('<%= value %>').default
+%>export const <%= key %> = () => import('<%= value %>').then(m => m.default)
```

Comment:
@rchl I couldn't find where is exactly used. We should update it to check function type and await on lazy import.

Reply:
31532e7fa74bc479704cac21ff4960fbd676bce6